spec.emu

```diff
@@ -98,20 +98,17 @@ contributors: Google, Ecma International
       The DisplayNames constructor is a standard built-in property of the Intl object.</p>
 
     <emu-clause id="sec-Intl.DisplayNames">
-      <h1>Intl.DisplayNames ([ _locales_ [ , _options_ ]])</h1>
+      <h1>Intl.DisplayNames ( _locales_ , _options_ )</h1>
 
       <p>
-        When the *Intl.DisplayNames* function is called with optional arguments the following steps are taken:
+        When the *Intl.DisplayNames* function is called with arguments the following steps are taken:
```

Comment:
Nit: Add the argument names:
```
When the *Intl.DisplayNames* function is called with arguments _locales_ and _options_, the following steps are taken: 
```


(The argument names are also missing in <https://tc39.es/ecma402/#sec-Intl.RelativeTimeFormat>, but let's not repeat the same issue here.)